server/Auth.js

```diff
@@ -173,6 +182,89 @@ class Auth {
     })
   }
 
+  /**
+   * Checks for an environment variable of PROXY_FORWARD_AUTH_ENABLED and if enabled
+   * will authenticate the user from the proxy headers.
+   * @param {*} req 
+   * @param {Array} feeds
+   * @returns {Promise<(Object|false)>} User login response payload or false
+   */
+  async getUserFromProxyAuth(req, feeds) {
+    if (!global.ForwardAuth.Enabled) {
+      return false
+    }
+
+    let username = req.headers[global.ForwardAuth.UsernameHeader]
+
+    if (!username) {
+      Logger.warn(`[Auth] Forward Auth username header ${global.ForwardAuth.UsernameHeader} has no username`)
+      return false
+    }
+
+    let user = this.users.find(u => u.username.toLowerCase() === username)
```

Comment:
I noticed just another critical bug. 

If you login with Browser A, everything works. If you then use the same account with Browser B, ANOTHER account with the same name is created. That happens when you have an account with an Uppercase letter

I think the error is here; `username` on the rightside also needs to be lowercased.